snapcraft/internal/lifecycle/_runner.py

```diff
@@ -59,6 +59,8 @@ def execute(step, project_options, part_names=None):
     :returns: A dict with the snap name, version, type and architectures.
     """
     config = project_loader.load_config(project_options)
+    if project_options.info.vendoring:
+        logger.warning('Vendoring should be used with a LXD container')
```

Comment:
We could probably make this a bit more helpful. WHY do we recommend a LXD container to be used? What side effects will NOT using a LXD container introduce?